config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
For the file name, maybe `versions.json` would be more descriptive?

Or even `dependencies.json`?